k8s.gcr.io/images/k8s-staging-sig-storage/generate.sh

```diff
@@ -37,7 +37,7 @@ readonly images=(
     mock-driver
     nfs-provisioner
     nfs-subdir-external-provisioner
-    # TODO: nfsplugin?
+    nfsplugin
```

Comment:
Please include the new generated file in this PR.

Reply:
do you mean I should add image tags to `k8s.gcr.io/images/k8s-staging-sig-storage/images.yaml`?

Reply:
I got failure when I run `generate.sh`, and I don't have a gcloud account:
```
k8s.gcr.io/images/k8s-staging-sig-storage/generate.sh: line 51: gcloud: command not found
```

I am following similar PR as https://github.com/kubernetes/k8s.io/pull/1829, do I have to run `generate.sh`?

cc @msau42

Reply:
IMHO it is better to keep images.yaml up-to-date, in particular when there are images that are ready to be promoted. That seems to be the case here (v3.0.0, v3.1.0).